oshmem/mca/spml/ucx/spml_ucx.h

```diff
@@ -130,7 +131,7 @@ mca_spml_ucx_get_mkey(int pe, void *va, void **rva)
 
 static inline int ucx_status_to_oshmem(ucs_status_t status)
 {
-    return OPAL_LIKELY(UCS_OK == status) ? OSHMEM_SUCCESS : OSHMEM_ERROR;
+    return OPAL_LIKELY(status >= 0) ? OSHMEM_SUCCESS : OSHMEM_ERROR;
```

Comment:
@shamisp IMHO we should change this only for non-blocking calls, by itroducing another function e.g ucx_status_to_oshmem_nb()